vevestaX/vevesta.py

```diff
@@ -828,3 +828,34 @@ def commit(self, techniqueUsed, filename=None, message=None, version=None, proje
             print("Wrote experiment to tool, Vevesta")
         else:
             print("Failed to write experiment to tool, Vevesta")
+
+    def git_commit(self, repoName, sourceBranch='main', newBranch=None, commitMessage=None):
```

Comment:
git commit will be private function

Reply:
git commit will be called from inside commit function

Reply:
It wont ask for  new branch name as argument, instead use the technique used appended with some 6 random numbers at the end.